site/en/docs/crux/methodology/index.md

```diff
@@ -0,0 +1,567 @@
+
+Pages and origins will be automatically included or removed from the dataset if their eligibility changes over time. There is not currently a way to manually submit pages or origins for inclusion.
+
+### User {: #user-eligibility}
+
+For a **user** to have their experiences aggregated in the CrUX dataset, they must meet the following criteria:
+
+1. Enable [usage statistic reporting](https://support.google.com/chrome/answer/96817).
+2. Sync their [browser history](https://support.google.com/chrome/answer/185277).
+3. Not have a [Sync passphrase](https://support.google.com/chrome/answer/165139?co=GENIE.Platform%3DAndroid#zippy=%2Ccreate-a-passphrase) set.
+4. Use a supported platform.
+
+The current supported platforms are:
+
+- Desktop versions of Chrome including Windows, MacOS, ChromeOS, and Linux operating systems.
+- Android versions of Chrome, including native apps using [Custom Tabs](/docs/android/custom-tabs/) and [WebAPKs](https://web.dev/webapks/).
+
+There are a few notable exceptions that do not provide data to the CrUX dataset:
+
+- Chrome on iOS.
+- Native Android apps using WebView.
+- Other Chromium browsers (for example [Microsoft Edge](https://www.microsoft.com/edge)).
+
+Chrome does not publish data about the proportions of users that meet these criteria. You can learn more about the data we collect in the [Chrome Privacy Whitepaper](https://www.google.com/chrome/privacy/whitepaper.html#usagestats).
+
+### Origin {: #origin-eligibility }
+
+An [**origin**](https://developer.mozilla.org/docs/Glossary/Origin) represents an entire website, addressable by a URL like `https://www.example.com`. For an origin to be included in the CrUX dataset it must meet two requirements:
+
+1. Publicly discoverable
+2. Sufficiently popular
+
+An origin is considered to be publicly discoverable if its **root page** is discoverable. For example, the root page of the origin `https://www.example.com` would have a URL of `https://www.example.com/`. If the root page has any HTTP redirects, the origin will be assessed based on the redirected page as experienced by real users.
+
+Any page will **not** meet the discoverability requirement if **any** of the following conditions are met, including root pages for the origin dataset:
+
+- The page is served with an HTTP [status code](https://developer.mozilla.org/docs/Web/HTTP/Status) other than `200`.
+- The page is served with an HTTP `X-Robots-Tag: noindex` [header](https://developers.google.com/search/docs/advanced/robots/robots_meta_tag#xrobotstag-implementation).
+- The document includes a `<meta name="robots" content="noindex">` [meta tag](https://developers.google.com/search/docs/advanced/robots/robots_meta_tag).
```

Comment:
Is this new? I remember that John Mueller stated that even noindex pages could be added..

Reply:
Not sure what John was referring to. An origin is not public if its root page is not discoverable. If an origin is public, it may contain aggregated data from all underlying pages, public _or_ private. We go into more detail on this a couple of paragraphs down.